app/src/main/java/com/example/marvelcomicsapp/util/Constants.kt

```diff
@@ -8,8 +9,8 @@ class Constants {
 
     companion object {
         const val BASE_URL = "https://gateway.marvel.com/v1/public/"
-        const val PUBLIC_KEY = "615340260e5f8063dd68d61dd8b676e2"
-        const val PRIVATE_KEY = "d288984e618e4e54177df9f44d52ed76f819157e"
+        const val PUBLIC_KEY =  BuildConfig.PUBLIC_KEY
+        const val PRIVATE_KEY =  BuildConfig.PRIVATE_KEY
```

Comment:
You don't need anymore those const values. You can use BuildConfig fields directly in code.